nautobot/extras/templates/extras/inc/panel_changelog.html

```diff
@@ -20,7 +20,7 @@
                 {% endif %}
                 <br />
                 <small>
-                    <span class="text-muted">{{ change.user|default:change.user_name }} -</span>
+                    <span class="text-muted">{{ change.user_name }} -</span>
```

Comment:
The odds that the latest 15 ObjectChange records are from a user who changed their username are?

Reply:
-0